packages/ethereum-contracts/test/foundry/agreements/ConstantFlowAgreementV1.prop.sol

```diff
@@ -68,4 +92,36 @@ contract ConstantFlowAgreementV1Properties is Test {
         uint256 deposit = cfa.getDepositRequiredForFlowRatePure(minimumDeposit, liquidationPeriod, flowRate);
         assert(deposit >= minimumDeposit);
     }
+
+    function testFlowDataEncoding(uint32 timestamp, int96 flowRate, uint64 depositClipped, uint64 owedDepositClipped)
+        public view
+    {
+        ConstantFlowAgreementV1.FlowData memory a = ConstantFlowAgreementV1.FlowData({
+            timestamp: uint256(timestamp),
+                    flowRate: flowRate,
+                    deposit: uint256(uint64(depositClipped << 32)),
+                    owedDeposit: uint256(uint64(owedDepositClipped << 32))});
+        uint256 wordA = cfa.encodeFlowData(a);
+        bool exist;
+        ConstantFlowAgreementV1.FlowData memory b;
+        (exist, b) = cfa.decodeFlowData(wordA);
+        assert(a.timestamp == b.timestamp);
+        assert(a.flowRate == b.flowRate);
+        assert(a.deposit == b.deposit);
+        assert(a.owedDeposit == b.owedDeposit);
+    }
+
+    function testFlowOperatorDataEncoding(uint8 permissions, int96 flowRateAllowance) public {
+        vm.assume(flowRateAllowance > 0);
```

Comment:
in `_encodeFlowOperatorData`, it checks for >= 0

Reply:
I will make a quick change

Reply:
resolved